README.md

```diff
@@ -23,47 +23,34 @@ programmatic interface
 3. Dissemination of the diverse information in a hierarchical digestible way
 for interpreting variants and patients.
 
-## Pre-requisites
-
-#### Mongo DB
-
-##### Alternative 1 - install mongoDB
-Install mongoDB manually.
-
-##### Alternative 2 - start mongoDB in a docker container (assuming genome-nexus is running locally, development mode)
-Start with docker:
-```
-docker network create gn-net
-docker run --name=gn-mongo --restart=always --net=gn-net -p 27017:27017 -d mongo
-```
-Enable the following line in `application.properties`:
-```
-spring.data.mongodb.import.command.prefix=docker run --rm --net=gn-net -v ${user.dir}:${user.dir} -v /tmp:/tmp mongo mongoimport --uri mongodb://gn-mongo:27017/annotator
-```
-##### Alternative 3 - start genome-nexus and mongoDB in docker containers (production mode)
-Create common docker network:
-```
-docker network create gn-net
-```
-Run your genome-nexus container using the network above (`--net` parameter in docker).
+## Run
 
-Run the mongoDB container:
+### Alternative 1 - run genome-nexus and mongoDB in docker containers
+Run with docker (assumes mvn installed locally):
 ```
-docker run --name=gn-mongo --restart=always --net=gn-net -d mongo
+mvn  -DskipTests clean install
+docker compose up --build
 ```
-**Only** change the following in `application.properties` (so do **not** enable the property used in Alternative 2 above):
+### Alternative 2 - run genome-nexus locally, but mongoDB in docker container
 ```
-spring.data.mongodb.uri=mongodb://gn-mongo:27017/annotator
+# change latest to different version if necessary (only need to run this once)
+docker run --name=gn-mongo --restart=always -p 27017:27017 -d genomenexus/gn-mongo:latest 
+mvn  -DskipTests clean install
+java -jar web/target/web-*.war
 ```
+### Alternative 3 - install mongoDB locally and run with local java
+Install mongoDB manually. Then follow instructions in
+[genome-nexus-importer](https://github.com/genome-nexus/genome-nexus-importer)
```

Comment:
following these instructions applies to all alternatives, right?

Reply:
@pieterlukasse actually it does not, because when you use docker compose, you don't have to initialize the database. It just pulls  mongo with the data imported

Reply:
Maybe I have to clarify in docs

Reply:
clarified